.github/workflows/ci.yml

```diff
@@ -98,21 +105,18 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: set up python
-        uses: actions/setup-python@v5
+      - name: set up uv
+        uses: astral-sh/setup-uv@v5
         with:
           python-version: '3.10'
 
-      - name: install
-        run: pip install -U build
-
       - name: check GITHUB_REF matches package version
-        uses: samuelcolvin/check-python-version@v3.1
+        uses: samuelcolvin/check-python-version@v4.1
         with:
           version_file_path: pydantic_settings/version.py
 
       - name: build
-        run: python -m build
+        run: uv build
```

Comment:
Let's not change the `release` part. as we are using the same thing in pydantic.